virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="virt-managing-vms-openshift-pipelines"]
+= Managing virtual machines with OpenShift Pipelines
+include::_attributes/common-attributes.adoc[]
+:context: virt-managing-vms-openshift-pipelines
+
+toc::[]
+
+xref:../../cicd/pipelines/understanding-openshift-pipelines.adoc#understanding-openshift-pipelines[Red Hat OpenShift Pipelines] is a Kubernetes-native CI/CD framework that allows developers to design and run each step of the CI/CD pipeline in its own container.
+
+The Tekton Tasks Operator (TTO) integrates {VirtProductName} with OpenShift Pipelines. TTO includes cluster tasks and example pipelines that allow you to:
+
+* Create and manage virtual machines (VMs), persistent volume claims (PVCs), and data volumes
+* Execute commands in VMs
+* Manipulate disk images with `libguestfs` tools
+
+:FeatureName: Managing virtual machines with OpenShift Pipelines
+include::snippets/technology-preview.adoc[]
+
+
+[id="prerequisites_virt-managing-vms-openshift-pipelines"]
+== Prerequisites
+
+* You have access to an {product-title} cluster with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+* You have xref:../../cicd/pipelines/installing-pipelines.adoc#installing-pipelines[installed OpenShift Pipelines].
+
+
+include::modules/virt-deploying-tto.adoc[leveloffset=+1]
+
+include::modules/virt-supported-tekton-tasks.adoc[leveloffset=+1]
+
+
+[id="example-pipelines_virt-managing-vms-openshift-pipelines"]
+== Example pipelines
+
+The Tekton Tasks Operator includes the following example `Pipeline` manifests. You can run the example pipelines by using the web console or CLI.
+
+Windows 10 installer pipeline:: This pipeline installs Windows 10 into a new data volume from a Windows installation image (ISO file). The installation process is controlled and executed by a custom answer file.
```

Comment:
Total nit: "is controlled and executed" seems potentially redundant. I am not too worried about the passive voice here, since it makes it less anthropomorphic, but I'd suggest tweaking this sentence. Maybe:

A custom answer file is used to run the installation process.